### ISSUE TITLE
Búsqueda

### ISSUE THREAD

**Issue by fvilledary** [DemocraciaEnRed/vos-que-propones#4]
Se planea hacer una busqueda tipo Instant que mientras vos tipeas te vaya cargando abajo todo lo que encuentre relacionado ya sean partidos, candidatos o propuestas.
Se puede hacer simplemente por contenido (ej: si buscas 'Frente', te aparecen todos los partidos que en su nombre tengan 'Frente' y así con los candidatos y propuestas también), ó a partir de labels, como que si buscas 'Salud' te aparezcan todas las propuestas de salud.

**Comment by jfresco** [comment]
En DemocracyOS usamos [List.js](http://www.listjs.com/), que además sirve para filtrar y ordenar.

**Comment by juandimaio** [comment]
Muy bueno el aporte @jfresco, estuve viendo los de [wired](http://www.wired.com/), del [pdr](http://partidodelared.org/) mismo y de [myspace](https://myspace.com/), esos tienen buscadores similares.

**Comment by gvilarino** [comment]
Con esto resuelven el autocomplete (i.e.: dropdown reactivo tipo FB) https://github.com/matthewmueller/autocomplete

**Comment by martinvol** [comment]
@gvilarino Es mas recomendable que el de jQueryUI?

**Comment by gvilarino** [comment]
No, es sólo una opción si no están usando `jQuery` y tienen que buscar en data del backend. Si tenés toda la data ya en el browser probablemente algo como lo que proponés tenga más sentido.

**Comment by martinvol** [comment]
Claro, toda la data está en el frontend al cargarse.

**Comment by gvilarino** [comment]
Entonces ni lo duden, `jQuery` va como piña. Rcuerden requierir el minificado ;)

**Comment by fvilledary** [comment]
Si, estamos con el minificado actualmente.

**Comment by gvilarino** [comment]
:+1:
